app/src/main/java/com/antosik/benchproject/app/popular/movies/viewModel/PopularMoviesViewModel.kt

```diff
@@ -38,23 +37,8 @@ class PopularMoviesViewModel @Inject constructor(
     private fun loadPopularMovie() {
         viewModelScope.launch {
             _isLoaderVisible.value = false
-            getPopularMoviesUseCase().fold(
-                onSuccess = { moviesModel ->
-                    _isLoaderVisible.value = true
-                    _popularMovies.value = moviesModel.toUi()
-                },
-                onFailure = {
-                    _isLoaderVisible.value = true
-                    popularMoviesFragmentNavigator.errorSnackBar(
-                        R.string.errorMessageMovies,
-                        onAction = { retryLoadPopularMovie() }
-                    )
-                }
-            )
+            _popularMovies.value = getPopularMoviesUseCase().toUi()
+            _isLoaderVisible.value = true
```

Comment:
Should the loader be visible? At this point the data should be ready I guess?

Reply:
Yes, the loader shouldn't be visible I will delete it